docs/api/spec/treetracker-wallet-api.yaml

```diff
@@ -1,10 +1,18 @@
 openapi: 3.0.0
 info:
-  title: treetracker token api
+  title: Treetracker Wallet API
   contact: {}
-  version: '0.9'
+  version: '1.27.3'
```

Comment:
Is it possible to have the version updated automatically as changes come in @pranavkparti

Reply:
Doesn't seem to be any straightforward way unfortunately. We might need to add a new action to the Github workflow to run some custom script.

Reply:
Can you look into that?